lib/panel/component.js

```diff
@@ -7,18 +7,37 @@ import type Delegate from './delegate'
 import type { LinterMessage } from '../types'
 
 class PanelComponent extends React.Component {
-  props: {
-    delegate: Delegate,
-  };
-  state: {
-    messages: Array<LinterMessage>,
-  };
+  static renderRowColumn(row: LinterMessage, column: string): string | Object {
+    const range = $range(row)
+
+    switch (column) {
+      case 'file':
+        return getPathOfMessage(row)
+      case 'line':
+        return range ? `${range.start.row + 1}:${range.start.column + 1}` : ''
+      case 'excerpt':
+        if (row.version === 1) {
+          if (row.html) {
+            return <span dangerouslySetInnerHTML={{ __html: row.html }} />
+          }
+          return row.text || ''
+        }
+        return row.excerpt
+      case 'severity':
+        return severityNames[row.severity]
+      default:
+        return row[column]
+    }
+  }
   constructor(props: Object, context: ?Object) {
     super(props, context)
     this.state = {
       messages: this.props.delegate.filteredMessages,
     }
   }
+  state: {
```

Comment:
This part of the PR conflicts with flow update PR (which I think should be merged first)

Reply:
It's actually not a conflict, this is a move due to an ESLint rule.